native/cocos/renderer/pipeline/custom/LayoutGraphGraphs.h

```diff
@@ -1068,8 +1068,8 @@ locate(std::string_view absolute, const LayoutGraph& g) noexcept {
 
 inline LayoutGraph::vertex_descriptor
 locate(LayoutGraph::vertex_descriptor u, std::string_view relative, const LayoutGraph& g) {
-    CC_EXPECTS(!boost::algorithm::starts_with(relative, "/"));
-    CC_EXPECTS(!boost::algorithm::ends_with(relative, "/"));
+    CC_EXPECTS(relative.substr(0, 1) != "/");
+    CC_EXPECTS(relative.empty() || relative.substr(relative.size() - 1) != "/");
```

Comment:
`boost::algorithm::string` and `boost::span` conflicts with graph code, remove `boost::algorithm::string`.